Sources/TootSDK/TootClient/TootClient+Request.swift

```diff
@@ -38,6 +38,16 @@ extension TootClient {
         if let offset = offset {
             queryParameters.append(.init(name: "offset", value: String(offset)))
         }
+        
+        if onlyMedia == true {
+            queryParameters.append(.init(name: "only_media", value: "true"))
+        }
```

Comment:
Unlikely, but if default value changed it would be impossible to pass false. I suggest always adding this param when it's not nil.

Reply:
Good catch, I’ll fix that there and also for the local/remote filtering.